### ISSUE TITLE
Installing your package

### ISSUE THREAD

**Issue by DocOfi** [Robinlovelace/geocompr#173]
Hi,

I tried installing your package using devtools::install_github("robinlovelace/geocompr")

Everything goes smoothly until it reaches the pacakge stars.  I then get this error:

"C:/PROGRA~1/R/R-34~1.1/bin/x64/R" --no-site-file --no-environ --no-save  \
  --no-restore --quiet CMD INSTALL  \
  "C:/Users/user/AppData/Local/Temp/RtmpYPKRAj/devtoolsb5872649dc/Robinlovelace-geocompr-d2a0a34"  \
  --library="C:/Users/user/Documents/R/win-library/3.4" --install-tests 

ERROR: dependency 'stars' is not available for package 'geocompr'
* removing 'C:/Users/user/Documents/R/win-library/3.4/geocompr'
Installation failed: Command failed (1)

I just finished the course Building data visualization tools from Coursera and i'm interested in making maps. Thank you.

**Comment by Robinlovelace** [comment]
That error message means that build this book you need to first install the **stars** package. You should be able to install it with the following command:

```r
devtools::install_github('r-spatial/stars')
```

I think this is a stars installation issue so closing for now but please let us know how you get on, any feedback on the book welcome.

**Comment by DocOfi** [comment]
Thanks for the quick reply.  I tried your suggestion and this is what the error I got:
* installing *source* package 'stars' ...
** R
** demo
** inst
** tests
** preparing package for lazy loading
Error : package 'sf' 0.6.0 was found, but >= 0.6.1 is required by 'stars'
ERROR: lazy loading failed for package 'stars'
* removing 'C:/Users/user/Documents/R/win-library/3.4/stars'
Installation failed: Command failed (1)

Since i got as far as this package, does that mean that all the other packages that was installed prior to stars were installed correctly? Are there any other packages after stars that needs to be loaded maybe i can make it work piecemeal.  I can try knitting the individual Rmd files which works without the stars package. Thanks again.
